### PR TITLE
gitsheet of your cheatsheets

### PR DESCRIPTION
**Pull request by Mauoke**
This is a file that sums up all the different cheatsheets present in your lessons (I mean, not really up to date right now, the last one is yet to be added).
I think it is more practical to have one document to go to, instead of having to navigate through all the lessons to find what we are looking for. There is still some improvements to make (like maybe indicating in which lesson you can find this for more explanations) but this is still an okay-ish base.

### CODE REVIEW

File: lessons/gitsheet.qmd [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+  title: Cheatsheet - git bash commands
+  author: Maureen
+  toc: true
+  title-block-banner: '#D09EBF'
```
Comment (reinhrst) [inline lessons/gitsheet.qmd:5]: I'm not sure about this colour

Comment (Mauoke) [reply lessons/gitsheet.qmd:5]: i like it

File: lessons/gitsheet.qmd [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  title-block-banner-color: 'black'
+---
+
+![a nerdy octopus, illustrated by @camerongraphics](images/octopus.jpg){fig-align="center" width="350"}
+
+::: {.callout-important appearance="simple"}
```
Comment (reinhrst) [inline lessons/gitsheet.qmd:11]: gfasgf sgdfg

File: lessons/gitsheet.qmd [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+This is a cheat sheet to go back to, when lost for some steps in git. It summaries all the cheat sheets present in the different lessons (yet to be updated).
+I did it for myself at the beginning, but you can take it and use it as it is or as a template for your own.
+
```
Comment (reinhrst) [inline lessons/gitsheet.qmd:14]: gsdgf sdgf sg sdg

File: lessons/gitsheet.qmd [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+To be filled in along my -our- journey with new tips/ commands.
+:::
+
+------------------------------------------------------------------------
+
+### basic commands
+
+```{bash}
+
+$ pwd          #show the current directory
```
Comment (reinhrst) [inline lessons/gitsheet.qmd:24]: gfdsgfdgsf

**Review by reinhrst — CHANGES_REQUESTED** [review]
These things wshould really be fixed before we can continue

File: lessons/gitsheet.qmd [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  author: Maureen
+  toc: true
+  title-block-banner: '#D09EBF'
+  title-block-banner-color: 'black'
+---
+
+![a nerdy octopus, illustrated by @camerongraphics](images/octopus.jpg){fig-align="center" width="350"}
+
+::: {.callout-important appearance="simple"}
+This is a cheat sheet to go back to, when lost for some steps in git. It summaries all the cheat sheets present in the different lessons (yet to be updated).
```
Comment (kristinpiening) [inline lessons/gitsheet.qmd:12]: 'it summarizes'

File: lessons/gitsheet.qmd [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+I did it for myself at the beginning, but you can take it and use it as it is or as a template for your own.
+
+To be filled in along my -our- journey with new tips/ commands.
```
Comment (PaulineBodson) [inline lessons/gitsheet.qmd:15]: Why is our cited like this -our-

**Review by kristinpiening — APPROVED** [review]

File: lessons/gitsheet.qmd (hunk 5 above)
Comment (AntoineGrissot) [inline lessons/gitsheet.qmd]: Great idea !

**Review by PaulineBodson — CHANGES_REQUESTED** [review]

**Review by AntoineGrissot — APPROVED** [review]

File: lessons/gitsheet.qmd (hunk 4 above)
Comment (kasiawojczulanis) [reply lessons/gitsheet.qmd:23]: Unclear comment

**Review by kasiawojczulanis — APPROVED** [review]

File: lessons/gitsheet.qmd (hunk 4 above)
Comment (reinhrst) [reply lessons/gitsheet.qmd:23]: Sorry!

File: lessons/gitsheet.qmd (hunk 5 above)
Comment (reinhrst) [reply lessons/gitsheet.qmd]: It still applies

**Review by reinhrst — APPROVED** [review]
Now I'm hgappy!

**Pull request merged**